extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -598,10 +785,15 @@ describe('amp-a4a', () => {
         a4a.buildCallback();
         a4a.preconnectCallback(false);
         const preconnects = doc.querySelectorAll('link[rel=preconnect]');
-        expect(preconnects.length).to.equal(2);
+        expect(preconnects.length).to.equal(3);
```

Comment:
Done.  When I looked it up on Chai, it appears that that use of `length` is deprecated and they suggest `lengthOf` instead: http://chaijs.com/api/bdd/#method_length